README.md

```diff
@@ -6,33 +6,37 @@ The Moby Project
-- Batteries included but swappable: Moby includes enough components to build fully featured container system, but its modular architecture ensures that most of the components can be swapped by different implementations.
+
+- Batteries included but swappable: Moby includes enough components to build a fully-featured container system, but its modular architecture ensures that most of the components can be swapped by different implementations.
+
 - Usable security: Moby provides secure defaults without compromising usability.
-- Developer focused: The APIs are intended to be functional and useful to build powerful tools.
-They are not necessarily intended as end user tools but as components aimed at developers.
-Documentation and UX is aimed at developers not end users.
+
+- Developer-focused: The APIs are intended to be functional and useful to build powerful tools.
+
+  They are not necessarily intended as end-user tools but as components aimed at developers.
+  Documentation and UX are aimed at developers not end-users.
 
 ## Audience
 
-The Moby Project is intended for engineers, integrators and enthusiasts looking to modify, hack, fix, experiment, invent and build systems based on containers.
+The Moby Project is intended for engineers, integrators, and enthusiasts looking to modify, hack, fix, experiment, invent and build systems based on containers.
 It is not for people looking for a commercially supported system, but for people who want to work and learn with open source code.
 
 ## Relationship with Docker
 
 The components and tools in the Moby Project are initially the open source components that Docker and the community have built for the Docker Project.
 New projects can be added if they fit with the community goals. Docker is committed to using Moby as the upstream for the Docker Product.
-However, other projects are also encouraged to use Moby as an upstream, and to reuse the components in diverse ways, and all these uses will be treated in the same way. External maintainers and contributors are welcomed.
+However, other projects are also encouraged to use Moby as an upstream and to reuse the components in diverse ways, and all these uses will be treated in the same way. External maintainers and contributors are welcomed.
 
 The Moby project is not intended as a location for support or feature requests for Docker products, but as a place for contributors to work on open source code, fix bugs, and make the code more useful.
-The releases are supported by the maintainers, community and users, on a best efforts basis only, and are not intended for customers who want enterprise or commercial support; Docker EE is the appropriate product for these use cases.
+The releases are supported by the maintainers, community, and users, on a best efforts basis only, and are not intended for customers who want an enterprise or commercial support; Docker EE is the appropriate product for these use cases.
```

Comment:
The `an` doesn't look correct here; "users who want enterprise or commercial support" is correct.

That said, `Docker EE` no longer exists, so we should probably remove those bits from this file.